draft/powder_data_presentation.md

```diff
@@ -100,6 +102,18 @@ which they appear. For example, only the measured points for the
 diffractogram identified by `_pd_diffractogram.id` should appear
 in the same data block.
 
+### Identifying related data blocks
+
+Data blocks belonging to the same dataset are distinguished by having
+identical values for the `_audit_dataset.id` data name. Values should
+be universally unique. Sources of universally unique values include
+the original powder block id generation algorithm and 
+[RFC9562](https://datatracker.ietf.org/doc/html/rfc9562) as implemented
```

Comment:
I am very in favour of using UUIDs. However, we should probably explicitly state that the `_audit_dataset.id` values are case-sensitive unlike UUIDs. Alternatively, we could redefine `_audit_dataset.id` as case-insensitive, although, I am unsure is this would work well with the original powder block id generation algorithm (are those values case-sensitive)?

Reply:
Having case-sensitivity increases the uniqueness of UUIDs compared to the standard, I suppose, so it's not too much of a problem. I'll add some text.